addons/website/static/src/scss/website.scss

```diff
@@ -894,7 +894,7 @@ header {
                     padding-top: 15vh;
                     padding-bottom: 15vh;
                 }
-                text-align: center;
+                text-align: center !important;
```

Comment:
needed because a "text-right" class was recently added in the header "magazine" template